### ISSUE TITLE
Support for packages without host application.

### ISSUE THREAD

**Issue by michael-zimmermann** [cashapp/AccessibilitySnapshot#229]
It is not possible to run accessibility snapshot testing (using point-free ios-snapshotting) when creating swift packages. 
The snapshots crash with 
```
AccessibilitySnapshot/SnapshotTesting+Accessibility.swift:57: Fatal error: Accessibility snapshot tests cannot be run in a test target without a host application
```

It doesn't appear that the host application is necessary, at least with the point-free framework. Here is a snapshot where the default guard causing the fatal error is overridden.

<img src="https://github.com/user-attachments/assets/e46d5b0f-771b-4eb9-be29-1c7db6aaf7b1" width="300"/>

I propose, to be able to pass a variable to the snapshotting-strategy to avoid checking if a host application is available. The default value `true` of this new parameter would keep the old strategy working the way it did and cause no backwards compatibility problems. By orverriding this to `false`, accessibility snapshots could also be done for swift packages.

```
        assertSnapshot(
            of: view,
            as: .accessibilityImage(
                shouldRunInHostApplication: false
            )
        )
```

**Comment by szotp** [comment]
The behavior with host application is strange.

parseAccessibilityElements works just fine with UIHostingController, does not matter if hosting app is present or not

But with regular UIView, it returns nothing. Even with code as simple as
```swift
let label = UILabel()
label.text = "Hello"
XCTAssertEqual(label.accessibilityLabel, "Hello") // fails without host app
```
